api/v1alpha1/initialbuildpipelineselector_types.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	tektonapi "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+)
+
+// Defines boolean value in a string to have not set as well.
+// +kubebuilder:validation:Enum=true;false;True;False;1;0
+type BoolString string
+
+const (
+	Bool_true  BoolString = "true"
```

Comment:
per our slack exchange, I thought this was giving you problems @mmorhun on kustomize

did you sort that out, or is simply using string per my suggestion a change you still plan on making

Reply:
I decided to go another way and use a pointer to bool. Then we have `true`, `false` and `nil`.